spock/src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration name="weld-spock Test" status="INFO" monitorInterval="5" strict="true">
```

Comment:
What's the purpose of this config file?

Reply:
It is the same purpose as `junit5/src/test/resources/logging.properties`, just that is uses log4j instead of JUL.
For having some log output that is also easily configurable and suppresses the weld info level.
The difference to the JUL one is, that it also works properly if you run test through the IDE automatically.
For the JUL case it only works if you run the tests through Maven or if you manually configure the JUL logging config file in the run configuration.